### PR TITLE
Platform/getting started gs

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/platform/introduction/getting-started.md [hunk 1]
```diff
@@ -6,8 +6,37 @@ menu:
...
-- Install
+Before following the steps below, you'll need to [download and install](https://docs.influxdata.com/introduction/download-install/) each component of the TICK stack.
+
+## Understand how Telegraf writes data to InfluxDB
+
+Once Telegraf is installed and started, it will send system metrics to InfluxDB by default, which automatically creates a ‘telegraf’ database.
+
+The configuration file for Telegraf specifies where metrics come from and where they go (inputs and outputs). In this example, we'll focus on CPU data, which is one of the default system metrics generated by Telegraf. For this example, it is worth noting some relevant values:
+
+* `[agent]\interval` - declares the frequency at which system metrics will be sent to InfluxDB
```
Comment (sanderson) [inline content/platform/introduction/getting-started.md]: Use a `.` instead of a `\` between `[agent]` and `interval`:

`[agent].interval`

Comment (sanderson) [reply content/platform/introduction/getting-started.md]: Sentence needs a period.

File: content/platform/introduction/getting-started.md [hunk 2]
```diff
@@ -6,8 +6,37 @@ menu:
...
+
+For details about the configuration file, see [Getting started with Telegraf](https://docs.influxdata.com/telegraf/v1.7/introduction/getting-started/).
+
+## Query data in InfluxDB
+
+As reviewed above, Telegraf is sending system data, including CPU usage, to InfluxDB. There are two ways you can query your InfluxDB data:
+
+* [In Chronograf with the Data  Explorer.](https://docs.influxdata.com/chronograf/latest/guides/create-a-dashboard/#step-3-create-your-query) Use the builder to select from your existing data and allow Chronograf to format the query for you. Alternatively, manually enter and edit a query. You can move between using the builder and manually editing the query.
+* [Using the command line interface.](https://docs.influxdata.com/influxdb/v1.6/query_language/data_exploration/)
+
```
Comment (sanderson) [inline content/platform/introduction/getting-started.md]: Provide an example query to use here. The following will query cpu usage from the system and from the user over the last 30 minutes.

```sql
SELECT "usage_system","usage_user" FROM "telegraf"."autogen"."cpu" WHERE time > now() - 30m
```

File: content/platform/introduction/getting-started.md [hunk 3]
```diff
@@ -6,8 +6,37 @@ menu:
...
+## Visualize that data in a Chronograf dashboard
+
+Now that you've explored your data with queries, you can build a dashboard in Chronograf to visualize the data. For details, see [Create a dashboard](https://docs.influxdata.com/chronograf/latest/guides/create-a-dashboard) and [Using pre-created dashboards](https://docs.influxdata.com/chronograf/v1.6/guides/using-precreated-dashboards/).
+
```
Comment (sanderson) [inline content/platform/introduction/getting-started.md]: Just as a really quick "initial success" for new users, I think you should walk them through creating a dashboard and adding a cell using the same query as above. Just really simple, bare-bones steps, then telling them to get more information in the Chronograf docs.

File: content/platform/introduction/getting-started.md [hunk 4]
```diff
@@ -6,8 +6,37 @@ menu:
...
+## Create an alert in Kapacitor based on that data
+
+Since InfluxDB is running on `localhost:8086`, Kapacitor finds it during start up and creates several subscriptions on InfluxDB. These subscriptions tell InfluxDB to send all the data it receives from Telegraf to Kapacitor.
+
+For step-by-step instructions on how to set up an alert in Kapacitor based on your data, see [Creating Chronograf alert rules](https://docs.influxdata.com/chronograf/latest/guides/create-alert-rules/).
```
Comment (sanderson) [inline content/platform/introduction/getting-started.md]: Same thought as above. Giving a bare-bones example to give them an "initial success," then sending them off to the full Kapacitor docs that go into a lot more detail.

File: content/platform/introduction/getting-started.md (hunk 1 above)
Comment (noramullen1) [reply content/platform/introduction/getting-started.md]: Ah, there's a typo in the Getting Started guide for Kapacitor (it also uses a slash).

Comment (sanderson) [reply content/platform/introduction/getting-started.md]: We don't really have a defined pattern for how we actually do this, so maybe it should stay a `\` for now. `internal` is a setting underneath `[agent]`. In other instances in the docs, this structure is communicated using a `.`.

File: content/platform/introduction/getting-started.md [hunk 5]
```diff
@@ -6,5 +6,44 @@ menu:
...
+
+Once Telegraf is installed and started, it will send system metrics to InfluxDB by default, which automatically creates a ‘telegraf’ database.
+
+The configuration file for Telegraf specifies where metrics come from and where they go (inputs and outputs). In this example, we'll focus on CPU data, which is one of the default system metrics generated by Telegraf. For this example, it is worth noting some relevant values:
+
+* `[agent].interval` - declares the frequency at which system metrics will be sent to InfluxDB.
+* `[[outputs.influxdb]]` - declares how to connect to InfluxDB and the destination database, which is the default ‘telegraf’ database.
+* `[[inputs.cpu]]` - declares how to collect the system cpu metrics to be sent to InfluxDB. Enabled by default.
+
+For details about the configuration file, see [Getting started with Telegraf](https://docs.influxdata.com/telegraf/latest/introduction/getting-started/).
```
Comment (stevebang) [inline content/platform/introduction/getting-started.md]: I'd also mention the "Configuring Telegraf" page, which covers the configuration settings.

**Pull request merged**